ui/lib/apps/InstanceProfiling/translations/index.ts

```diff
@@ -0,0 +1,4 @@
+import zh from './zh.yaml'
+import en from './en.yaml'
+
+export default { zh, en }
```

Comment:
Could you extract this change to the master to make it a standard for all apps? Then new PRs from @shhdgit like https://github.com/pingcap/tidb-dashboard/pull/1092 will not break this PR even if it is merged before this PR.

Reply:
Good advice!